src/components/Skeleton/Skeleton.tsx

```diff
@@ -68,27 +67,31 @@ const SkeletonContainer = styled(Div)`
         }
     `
       : ''
-  }
+    }
   `}
 `;
 
 export type SkeletonProps = {
   StyledContainer?: StyledSubcomponentType;
   StyledShimmer?: StyledSubcomponentType;
 
-  containerProps?: SubcomponentPropsType;
+  SkeletonProps?: SubcomponentPropsType;
```

Comment:
I'd keep this name the same, as it matches the StyledContainer and other components where "container" is used for the outer-most element of the component.
```suggestion
  containerProps?: SubcomponentPropsType;
```